### PR TITLE
Update Quests.json - F2P Quests 2-5 added

### PR DESCRIPTION
**Pull request by Hannah-GBS**
Continues progress on #5

### CODE REVIEW

File: src/main/resources/Quests.json [hunk 1]
```diff
@@ -65,6 +66,7 @@
                   "itemQuantity":1
                }
             ],
+            "chunkLocation":null,
```
Comment (Little0smit) [inline src/main/resources/Quests.json]: Shouldn't this chunk location be the lumbridge chunk, as that's where you should be to complete the step?

File: src/main/resources/Quests.json [hunk 2]
```diff
@@ -76,6 +78,7 @@
...
```
Comment (Little0smit) [inline src/main/resources/Quests.json]: Shouldn't this chunk location be the lumbridge chunk, as that's where you should be to complete the step?

File: src/main/resources/Quests.json [hunk 3]
```diff
@@ -87,11 +90,13 @@
...
```
Comment (Little0smit) [inline src/main/resources/Quests.json]: Shouldn't this chunk location be the lumbridge chunk, as that's where you should be to complete the step?

File: src/main/resources/Quests.json [hunk 4]
```diff
@@ -87,11 +90,13 @@
...
             "extraInfo":""
          },
          {
             "stepNumber":5,
             "stepDescription":"Speak to Cook",
+            "itemReqs":null,
```
Comment (Little0smit) [inline src/main/resources/Quests.json]: @apan145 will this work okay for importing, or should this be handled differently?

File: src/main/resources/Quests.json [hunk 5]
```diff
@@ -127,5 +132,335 @@
       ],
       "questPointsReqs":0,
       "questPointsReward":1
+   },
```
Comment (Little0smit) [inline src/main/resources/Quests.json:91]: Need to add empty skill reqs here? @apan145 or will it work if the field is not there if not needed.

**Review by Little0smit — CHANGES_REQUESTED** [review]
Looks good, just a few points to clarify on format for importing, but seems great overall.

File: src/main/resources/Quests.json (hunk 1 above)
Comment (Hannah-GBS) [reply src/main/resources/Quests.json]: Fixed

File: src/main/resources/Quests.json (hunk 2 above)
Comment (Hannah-GBS) [reply src/main/resources/Quests.json]: Fixed

File: src/main/resources/Quests.json (hunk 3 above)
Comment (Hannah-GBS) [reply src/main/resources/Quests.json]: Fixed

File: src/main/resources/Quests.json (hunk 5 above)
Comment (Hannah-GBS) [reply src/main/resources/Quests.json:91]: It's there already? Above the quest point requirements.

Comment (Little0smit) [reply src/main/resources/Quests.json:91]: Yup, i'm blind/editor compressed it perfectly.

File: src/main/resources/Quests.json (hunk 4 above)
Comment (apan145) [reply src/main/resources/Quests.json]: The JSON is parsed just fine, but if we let itemReqs be set to null, we have to check for it. I think it is easier codewise to set itemReqs to an empty array if the step requires no items.

Comment (Little0smit) [reply src/main/resources/Quests.json]:
```suggestion
            "itemReqs":[],
```

Comment (Hannah-GBS) [reply src/main/resources/Quests.json]: There's a bunch of those I gotta change then, gimme a few

**Review by Little0smit — CHANGES_REQUESTED** [review]
Let's remove the dummy layout at the start, if you wan't to keep it for format guide add it to the jsonformat file.

**Review by Little0smit — APPROVED** [review]

**Pull request merged**